scripts/testnet/include/config.sh

```diff
@@ -8,6 +8,11 @@ generateConfig() {
     TMP_META_OBSERVERCOUNT=0
   fi
 
+  SOVEREIGN_BOOL="false"
+  if [ $USE_HARDFORK -eq 1 ]; then
+    SOVEREIGN_BOOL="true"
```

Comment:
Why set this to true if `USE_HARDFORK`?
This is wrong.
We need to use  `SOVEREIGN_DEPLOY`

Reply:
right